blazar/manager/service.py

```diff
@@ -692,7 +692,8 @@ def end_lease(self, lease_id, event_id):
         lease = self.get_lease(lease_id)
         allocations = self._existing_allocations(lease['reservations'])
         try:
-            self.enforcement.on_end(context.current(), lease, allocations)
+            self.enforcement.on_end(trusts.create_ctx_from_trust(
```

Comment:
Can you please leave a detailed comment above this line explaining why we have to use `create_ctx_from_trust` here?